UI/forms/MainToolBar.ui

```diff
@@ -0,0 +1,691 @@
+         <normaloff>:/res/images/sources/camera.svg</normaloff>:/res/images/sources/camera.svg</iconset>
+       </property>
+       <property name="flat">
+        <bool>true</bool>
+       </property>
+       <property name="themeID" stdset="0">
+        <string>cameraIcon</string>
+       </property>
+      </widget>
+     </item>
+     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
+      <widget class="QCheckBox" name="toolBarStudioMode">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="minimumSize">
+        <size>
+         <width>0</width>
+         <height>0</height>
+        </size>
+       </property>
+       <property name="toolTip">
+        <string>Basic.EnableStudioMode</string>
+       </property>
+       <property name="accessibleName">
+        <string>Basic.EnableStudioMode</string>
+       </property>
+       <property name="styleSheet">
+        <string notr="true">QCheckBox::indicator{subcontrol-position: center;}</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <spacer name="horizontalSpacer">
+       <property name="orientation">
+        <enum>Qt::Horizontal</enum>
+       </property>
+       <property name="sizeHint" stdset="0">
+        <size>
+         <width>40</width>
+         <height>20</height>
+        </size>
+       </property>
+      </spacer>
+     </item>
+     <item>
+      <widget class="MenuButton" name="streamButton">
+       <property name="toolTip">
+        <string>Basic.Main.StartStreaming</string>
+       </property>
+       <property name="accessibleName">
+        <string>Basic.Main.StartStreaming</string>
+       </property>
+       <property name="text">
+        <string>Basic.Main.GoLive</string>
+       </property>
+       <property name="icon">
+        <iconset resource="obs.qrc">
+         <normaloff>:/res/images/streaming-inactive.svg</normaloff>:/res/images/streaming-inactive.svg</iconset>
+       </property>
+       <property name="checkable">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QPushButton" name="broadcastButton">
+       <property name="text">
+        <string>Basic.Main.SetupBroadcast</string>
+       </property>
+       <property name="icon">
+        <iconset resource="obs.qrc">
+         <normaloff>:/res/images/sliders.svg</normaloff>:/res/images/sliders.svg</iconset>
+       </property>
+       <property name="checkable">
+        <bool>true</bool>
+       </property>
+       <property name="themeID" stdset="0">
+        <string>slidersIcon</string>
+       </property>
+      </widget>
+     </item>
+     <item alignment="Qt::AlignVCenter">
+      <widget class="QLabel" name="streamTimerLabel">
```

Comment:
I'm personally not a fan of moving the timers amongst buttons - it adds visual noise & complexity and makes it harder to check them at a glance.

Reply:
Updated to remove the control buttons/timers.